src/battle-animations.ts

```diff
@@ -693,17 +691,33 @@ class BattleScene {
 			if (i % 3 === 2) pokemonhtml += `</div><div class="teamicons">`;
 		}
 		pokemonhtml = '<div class="teamicons">' + pokemonhtml + '</div>';
+		const ratinghtml = side.rating ? ` title="Rating: ${BattleLog.escapeHTML(side.rating)}"` : ``;
+		let posStr = "";
+		if (isAlly) {
+			if (!side.isFar) {
+				posStr += "p3";
+			} else {
+				posStr += "p4";
+			}
+		} else {
+			posStr = !side.isFar ? "p1" : "p2";
+		}
```

Comment:
It sounds like we shouldn't be using `p1/p2/p3/p4` for these at all, considering they look like side IDs even though they aren't. Maybe `near`, `near2`, `far`, and `far2`?